### ISSUE TITLE
The command "whitelist show" doesn't support gamertags with spaces

### ISSUE THREAD

**Issue by UponAnonymous** [nxck69/DayZConsoleNitradoAdministrationBot#2]
As you can see when someone has a space in their gamertag like "Enderty D" the bot detects it as a new gamertag which then puts it on a new line.

![image](https://user-images.githubusercontent.com/27465814/143619460-b4e91776-6f77-47c0-ad00-ae3d5ce9c103.png)

**Comment by nxck69** [comment]
Yeah this Script is very outdated as of now. Will update this soon
